m_velychko/variables-and-types/message.py

```diff
@@ -4,6 +4,10 @@
 # Напишіть програму, яка виводить текстове повідомлення на екран. Змінна `message` вже містить текст повідомлення.
 # Ваше завдання - вивести це повідомлення за допомогою функції `print`.
 
+message = ("Привіт, це моя перша програма!")
```

Comment:
Cкобки вокруг строки можно не писать

```suggestion
message = "Привіт, це моя перша програма!"
```